examples/aws/main.tf

```diff
@@ -0,0 +1,48 @@
+provider kubermatic {
+  host = "set_it_up"
+}
+resource "kubermatic_project" "example_project" {
+  name = var.project_name
+}
+resource "kubermatic_cluster" "example_cluster" {
+  name       = var.cluster_name
+  dc_name    = "aws-eu-central-1a"
+  project_id = kubermatic_project.example_project.id
+  spec {
+    version = var.k8s_version
+    cloud {
+      aws {
+        access_key_id     = "set_it_up"
+        secret_access_key = "set_it_up"
+      }
+    }
+  }
+}
+resource "kubermatic_node_deployment" "example_node" {
+  name       = "examplenode"
+  cluster_id = kubermatic_cluster.example_cluster.id
```

Comment:
I think this example won't pass, as I saw the cluster_id format changed
https://github.com/kubermatic/terraform-provider-kubermatic/blob/master/kubermatic/resource_node_deployment.go#L31

Reply:
I've tested it. Works fine

Reply:
ok